src/applications/vaos/reducers/expressCare.js

```diff
@@ -126,7 +127,16 @@ export default function expressCareReducer(state = initialState, action) {
           facilityId: facility.id,
           days: facility.customRequestSettings
             .find(setting => setting.id === EXPRESS_CARE)
-            .schedulingDays.filter(day => day.canSchedule),
+            .schedulingDays.filter(day => day.canSchedule)
+            .map(daySchedule => ({
+              ...daySchedule,
+              dayOfWeekIndex: Number(
+                moment()
+                  .day(daySchedule.day)
+                  .format('d'),
+              ),
+            }))
+            .sort((a, b) => (a.dayOfWeekIndex < b.dayOfWeekIndex ? -1 : 1)),
```

Comment:
Adding day of week so we can more easily compare current day to days after.  Also sort schedule starting from Sunday since Sunday is designated as "0" by moment